src/lxc/conf.c

```diff
@@ -3521,9 +3520,9 @@ static bool verify_start_hooks(struct lxc_conf *conf)
 		if (ret < 0 || ret >= PATH_MAX)
 			return false;
 
-		ret = stat(path, &st);
+		ret = access(path, F_OK);
```

Comment:
Wouldn't X_OK actually be better since we'll be execing it?

Reply:
Hm, good point.

Reply:
updated